akamai/update_api.py

```diff
@@ -142,20 +143,20 @@ def activateVersion(version_number, env="STAGING"):
 
 def generateExclusions(frontend_path, config):
     exclusions = []
-    for app in (x for x in config if "frontend_paths" in config[x] and frontend_path not in config[x]["frontend_paths"]):
-        for path in config[app]["frontend_paths"]:
+    for key in (x for x in config.keys() if "frontend" in config[x] and "paths" in config[x]["frontend"] and frontend_path not in config[x]["frontend"]["paths"]):
```

Comment:
I want to use `for key, app in ()` but I'm not sure how to do that with the filtering I'm doing in the generator expression. Anyone know how to do that? I think it would look cleaner.

Reply:
I would consider pulling the python https://pypi.org/project/pylodash/ dep

Reply:
I did pull it in for a while to see if it would help. I couldn't find a way to use these functions to alleviate the issue, though. If you have an idea, can you please let me know?